gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidHandler.java

```diff
@@ -47,31 +33,28 @@
  */
 @Component
 public class CseValidHandler {
-    private final DichotomyRunner dichotomyRunner;
     private final EicCodesConfiguration eicCodesConfiguration;
     private final EicCodesMapper eicCodesMapper;
     private final FileImporter fileImporter;
     private final FileExporter fileExporter;
     private final Logger businessLogger;
-    private final CseValidNetworkShifter cseValidNetworkShifter;
-    private final CseValidRaoValidator cseValidRaoValidator;
+    private final ComputeFullImportService computeFullImportService;
+    private final ComputeExportCornerService computeExportCornerService;
```

Comment:
I would have called the new classes `FullImportComputationService` and `ExportCornerComputationService` rather than `ComputeFullImportService` and `ComputeExportCornerService`